app/Module.scala

```diff
@@ -14,13 +14,53 @@
  * limitations under the License.
  */
 
-import com.google.inject.AbstractModule
+import com.google.inject.{AbstractModule, Provides}
 import controllers.RemoteMessagesApiProvider
+import org.apache.pekko.stream.Materializer
 import play.api.libs.concurrent.PekkoGuiceSupport
+import play.api.{Configuration, Environment, Logger}
+import services._
+import uk.gov.hmrc.objectstore.client.play.PlayObjectStoreClient
+
+import javax.inject.Singleton
+import scala.concurrent.ExecutionContext
+
+class Module(env: Environment, playConfig: Configuration) extends AbstractModule with PekkoGuiceSupport {
 
-class Module() extends AbstractModule
-    with PekkoGuiceSupport {
   override def configure(): Unit = {
     bind(classOf[RemoteMessagesApiProvider])
+    bind(classOf[GovWalesCacheUpdateScheduler]).asEagerSingleton()
+  }
+
+  @Provides
+  @Singleton
+  private def provideWelshCountryNamesDataSource(english: EnglishCountryNamesDataSource,
```

Comment:
You may want to add a type annotation here so that there are no surprises.

Reply:
Have resolved this, please can you ✅ again @ssaleem-ee